src/styles/blocks/card-grid.scss

```diff
@@ -0,0 +1,101 @@
+@import "../utils/variables";
+@import "../utils/functions";
+@import "../utils/mixins";
+@import "../utils/extends";
+
+.page__catalog {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.product-grid {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
you don't need it, you use margins down below

Reply:
@jstmpelowycz but I use it only bottom, not on the right side of the cards: should I add one more margin instead?